index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <div class="starting-page">
+        <header class="starting-page>
+            <nav id="navigator_bar">
+                <a class="logo" href="#"><h2>Mylogo</h2></a>
+                <nav class="navigator_menu"><img src="assets/Enabled.png">
+                </nav>
+            </nav>
+        </header>
+        <section id="section-presentation">
+            <h4 class="short-presentation">I'm shri sakthi <br> glad to see you...</h4>
```

Comment:
- [ ] As requested by the previous reviewer please use the `h1` tag in place of the `h4` tag because If you want that google (and other search engines) know about a specific topic, in this case, you, use an h1 tag.

Reply:
Resolved the issue. Thank you for notifying me.